.devcontainer/Dockerfile

```diff
@@ -3,17 +3,14 @@
 ARG VARIANT="ubuntu-22.04"
 FROM mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
 
-# Setup MS package signing key:
-# NB: needs changing if VARIANT changes
-RUN wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb && \
-    dpkg -i packages-microsoft-prod.deb && \
-    rm packages-microsoft-prod.deb
-
-# Install APT HTTPS transport required for MS packages:
-RUN apt-get update && export DEBIAN_FRONTEND=noninteractive && apt-get install -y apt-transport-https
-
 # Install packages required for build:
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -y install --no-install-recommends \
     libunwind-dev clang build-essential libssl-dev pkg-config lldb \
-    bash-completion npm python-is-python3
+    bash-completion npm python-is-python3 \
+    dotnet6
+
+# Install Rust (installing via devcontainer feature failing for 1.63)
```

Comment:
Is there a bug worth linking?

Reply:
I didn't look too deeply into it; the script failed with an "exit 1" with no further error…